staging/src/k8s.io/apimachinery/pkg/runtime/scheme_test.go

```diff
@@ -978,46 +967,3 @@ func TestMetaValues(t *testing.T) {
 		t.Errorf("Expected %v, got %v", e, a)
 	}
 }
-
-func TestMetaValuesUnregisteredConvert(t *testing.T) {
```

Comment:
can you summarize why this test was removed? it looks like it's verifying a registered conversion function for an unregistered type still gets called. is that not what we still expect to happen?

Reply:
Good catch. I want to remove it once we stop supporting arbitrary conversions, but it's not now.
Reverted.